lib/phoenix/live_dashboard/info/process_info_component.ex

```diff
@@ -7,6 +7,7 @@ defmodule Phoenix.LiveDashboard.ProcessInfoComponent do
     :registered_name,
     :current_function,
     :initial_call,
+    :dictionary,
```

Comment:
Are the changes to this file necessary? We are not really rendering the dictionary, are we? :)

Reply:
No, the dictionary isn't rendered, but the `initial_call` is. So the dictionary must be fetched to display the "correct" `$initial_call` from the dictionary. Maybe the naming should be different (because not the dictionary is rendered, but the `initial_call` inside the dictionary), but that would make some greater changes to the component necessary.

Reply:
Hrm. `fetch_process_info` will never return the dictionary key because it is pre-processed, so I don't see why this line and the new `format_info` clauses are required. What am I missing? :)

Reply:
Probably I am missing something...
Doesn't `fetch_process_info` pass the `@info_keys` down to the `process_info_callback` (and therefore `Process.info`) in `SystemInfo`? 
I tested it with a fresh Phoenix application and it seemed to work.

Reply:
Ah, I see. I am the one getting confused. :) Thanks! :heart:

Yes, this is correct right now but we really don't want to send the dictionary, which can be quite large, across nodes.

So I think we can merge this as is and then refactor next!